### PR TITLE
Ownership Change

### PR DESCRIPTION
**Pull request by lparkermg**
# Ownership Transfer

As we are now working under Corsham Science, the package name, copyright information and other relevant metadata have been updated to reflect this fact.

# Package Updates

The dev packages used to build and test the `ViewModels.Core` have been updated to their latest version. This includes the `CodeStyle` package, which enforced much of the name change.

### CODE REVIEW

File: src/ViewModels.Core/Exceptions/EntityNotFoundException.cs [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+﻿// <copyright file="EntityNotFoundException.cs" company="Corsham Science">
+// Copyright (c) Corsham Science. All rights reserved.
 // </copyright>
 
-namespace CR.ViewModels.Core.Exceptions
+namespace CorshamScience.ViewModels.Core.Exceptions
 {
     using System;
     using System.Runtime.Serialization;
+    using CorshamScience.ViewModels.Core;
```
Comment (lparkermg) [inline src/ViewModels.Core/Exceptions/EntityNotFoundException.cs]: I don't think this needs to be added.

File: src/ViewModels.Core/Exceptions/DuplicateKeyException.cs [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+﻿// <copyright file="DuplicateKeyException.cs" company="Corsham Science">
+// Copyright (c) Corsham Science. All rights reserved.
 // </copyright>
 
-namespace CR.ViewModels.Core.Exceptions
+namespace CorshamScience.ViewModels.Core.Exceptions
 {
     using System;
     using System.Runtime.Serialization;
+    using CorshamScience.ViewModels.Core;
```
Comment (lparkermg) [inline src/ViewModels.Core/Exceptions/DuplicateKeyException.cs]: See above

File: src/ViewModels.Core/InMemoryViewModelRepository.cs [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
-namespace CR.ViewModels.Core
+namespace CorshamScience.ViewModels.Core
 {
     using System;
     using System.Collections.Concurrent;
     using System.Collections.Generic;
     using System.Linq;
     using System.Linq.Expressions;
-    using Exceptions;
+    using CorshamScience.ViewModels.Core.Exceptions;
```
Comment (lparkermg) [inline src/ViewModels.Core/InMemoryViewModelRepository.cs:12]: Does this need the full namespace?

File: src/ViewModels.Core/IViewModelWriter.cs [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
+// Copyright (c) Corsham Science. All rights reserved.
 // </copyright>
 
-namespace CR.ViewModels.Core
+namespace CorshamScience.ViewModels.Core
 {
     using System;
     using System.Linq.Expressions;
-    using Exceptions;
+    using CorshamScience.ViewModels.Core.Exceptions;
```
Comment (lparkermg) [inline src/ViewModels.Core/IViewModelWriter.cs:9]: See above.

File: src/ViewModels.Core/ViewModels.Core.csproj [hunk 5]
```diff
@@ -3,36 +3,44 @@
...
+    <Copyright>Corsham Science 2019</Copyright>
     <Description>A package containing .NET interfaces for reading/writing CQRS ViewModels in a generic way.</Description>
-    <PackageLicenseUrl>https://github.com/cognisant/ViewModels.Core/blob/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/cognisant/ViewModels.Core</PackageProjectUrl>
-    <PackageIconUrl>https://raw.githubusercontent.com/cognisant/ViewModels.Core/master/logo.png</PackageIconUrl>
-    <RepositoryUrl>https://github.com/cognisant/ViewModels.Core</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/qphl/ViewModels.Core</PackageProjectUrl>
+    <PackageIconUrl>https://raw.githubusercontent.com/qphl/ViewModels.Core/master/logo.png</PackageIconUrl>
+    <RepositoryUrl>https://github.com/qphl/ViewModels.Core</RepositoryUrl>
     <PackageTags>View Model, CQRS, RavenDB</PackageTags>
```
Comment (lparkermg) [inline src/ViewModels.Core/ViewModels.Core.csproj]: It would be worth removing the RavenDB tag since this specific project isn't tied to use in it.

**Review by lparkermg — COMMENTED** [review]
A few questions.

File: src/ViewModels.Core/InMemoryViewModelRepository.cs (hunk 3 above)
Comment (Calummullen) [reply src/ViewModels.Core/InMemoryViewModelRepository.cs:12]: It technically doesn't, but codestyles complains without it

**Review by lparkermg — COMMENTED** [review]
LGTM

**Review by ghost — APPROVED** [review]
Luke says it's good

**Pull request merged**